gc/base/standard/ConcurrentGC.hpp

```diff
@@ -381,10 +360,26 @@ class MM_ConcurrentGC : public MM_ParallelGlobalGC
 	ConHelperRequest getConHelperRequest(MM_EnvironmentBase *env);
 
 protected:
-	bool initialize(MM_EnvironmentBase *env);
+	virtual bool initialize(MM_EnvironmentBase *env);
```

Comment:
Don't make this virtual (it would be making like a C++ constructor virtual) - none or rarely they are virtual in the code base . At may be not as elegant, but at the call site you should specifically instantiate/initialize SATB or IU variant of this class. There actually probably only called by their specifc newInstance call, so just even removing 'virtual' might work!?

Reply:
removed virtual.